modules/persistent-storage-local-create-machine-config.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-local.adoc
+
+[id="local-create-machine-config_{context}"]
+= Configure an additional storage device to use local volumes
+
+You can configure additional storage devices for local volumes by defining new machineconfig resources.
+
+.Prerequisites
+
+* The Local Storage Operator is installed.
+* Local disks are attached to the {product-title} nodes.
```

Comment:
@bobfuru These prerequisites look good to me, but I don't understand the procedure.
IIUC, if we know the local disks filepath in the nodes(such as /dev/vda, /dev/xvda, I don't think there is some difference between Coreos and RHEL), we can just add the local disks filepath to the "localvolume" object or create a new "lcoalvolume" object, then the PVs for these local disks will be created automatically.

Reply:
Thanks! I have revised based on your suggestion and the BZ requestor's. PTAL at my updates.